### ISSUE TITLE
`pause` is not supported in Alda 2, but `note` is

### ISSUE THREAD

**Issue by UlyssesZh** [alda-lang/alda#382]
Something like
```
alda play -c "piano: (note (pitch '(a sharp)) (duration (ms 1000)))"
```
can be played without error, but in
```
alda play -c "piano: (pause (duration (ms 1000)))"
```
the `pause` is an unresolvable symbol.

**Comment by daveyarwood** [comment]
Good call! I think we should add `pause` to help with backwards compatibility.

(BTW, I appreciate all the issues you've been filing, I'm sorry I haven't had time to respond to them yet! I hope to spend some time doing that this weekend.)

**Comment by ManasviGoyal** [comment]
hi. can you pls elaborate ? would like to work on this

**Comment by UlyssesZh** [comment]
> hi. can you pls elaborate ? would like to work on this

You can refer to the `pause` function in Alda 1.

**Comment by daveyarwood** [comment]
@ManasviGoyal The `pause` function returns a [rest](https://en.wikipedia.org/wiki/Rest_(music)). There are two arities:

```lisp
;; equivalent to "r", i.e. a rest with the last specified duration
(pause)

;; equivalent to "r4", i.e. a quarter note rest
(pause (note-length 4))
```

If you're interested in implementing this, have a look at how [the `note` function](https://github.com/alda-lang/alda/blob/85d6acab6e930d10bc733722e1c0f8b39d6ebfce/client/model/lisp.go#L1276-L1297) is implemented in lisp.go, as a starting point.

I'd be happy to help if you get stuck! Feel free to reach out either here in the issue, or in the #development channel in [Alda Slack](https://slack.alda.io).

**Comment by JustinLocke** [comment]
Hi, 

Is this still an issue in alda 2? I would be willing to work on this task if it is still open.

**Comment by daveyarwood** [comment]
@JustinLocke Yes, `pause` is still not implemented. You're welcome to work on it, that would be great!

**Comment by daveyarwood** [comment]
Implemented in #413 (Thanks @JustinLocke!) I will include this in a new release soon, hopefully today!

**Comment by daveyarwood** [comment]
Fixed in Alda 2.2.1.
